CONTRIBUTING.md

```diff
@@ -79,13 +79,17 @@ Helpful issues usually include:
 
 Contributions to the documentation site are made by editing the docs repository. You can do this directly on GitHub.com or by creating a copy of the repository locally, making your changes there, and contributing back to our repository.
 
+> **Note**
+> By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title and/or in a commit message.  See the next section for more details.
```

Comment:
```suggestion
> By default, your merged PR to an English page will trigger our Translation Status Tracker. If your change should **NOT** be applied to every language (e.g. a typo fix to an English word), please include the word "minor" in your PR title.  See the next section for more details.
```

I'm changing to this, but I'm making a note that after using this for a while, people are often making "minor" PRs that are small, but *should* trigger i18n updates. I'm thinking that we were being too clever with this word choice, and might prefer we think of a word that is actually related to needing a translation or not.